app/renderer/components/common/messageBox.js

```diff
@@ -154,7 +158,10 @@ const styles = StyleSheet.create({
     marginTop: globalStyles.spacing.dialogInsideMargin,
     minWidth: '425px',
     marginBottom: globalStyles.spacing.dialogInsideMargin,
-    userSelect: 'text'
+    userSelect: 'text',
+    maxHeight: '300px',
```

Comment:
what if we would apply `vh` instead of `px`? 300px might be small for full HD monitors.